packages/wonder-blocks-modal/components/one-pane-dialog/one-pane-dialog.md

```diff
@@ -93,46 +96,165 @@ const styles = StyleSheet.create({
         top: 0,
         bottom: 0,
     },
+});
 
-    customModal: {
-        maxWidth: 400,
-        maxHeight: 400,
+const defaultStyles = StyleSheet.create({
+    row: {
+        flexDirection: "row",
+        justifyContent: "flex-end"
     },
+    button: {
+        marginRight: 16
+    }
+});
 
-    above: {
-        background: "url(/modal-above.png)",
-        backgroundSize: "cover",
-        width: 630,
-        height: 397,
-        position: "absolute",
-        top: 40,
-        left: -120
+const smallStyles = StyleSheet.create({
+    row: {
+        flexDirection: "column-reverse",
+        width: "100%"
     },
+    button: {
+        marginBottom: 16
+    }
 });
 
+const styleSheets = {
+    mdOrLarger: defaultStyles,
+    small: smallStyles
+};
+
+const Footer = () => (
+    <MediaLayout styleSheets={styleSheets}>
+        {({ styles }) => (
+            <View style={styles.row}>
+                <Button style={styles.button} kind="tertiary">
+                    Tertiary action
+                </Button>
+                <Button style={styles.button} kind="tertiary">
+                    Secondary action
+                </Button>
+                <Button style={styles.button}>Primary action</Button>
+            </View>
+        )}
+    </MediaLayout>
+ );
+
 <View style={styles.previewSizer}>
     <View style={styles.modalPositioner}>
         <OnePaneDialog
-            title="Custom modal"
-            subtitle="With long content"
+            title="Multi-line title that wraps to show how this component adjusts with longer content"
+            breadcrumbs={
+                <Breadcrumbs>
+                    <BreadcrumbsItem>
+                        Bread
+                    </BreadcrumbsItem>
+                    <BreadcrumbsItem>
+                        Crumb
+                    </BreadcrumbsItem>
+                    <BreadcrumbsItem aria-current="page">Trail</BreadcrumbsItem>
```

Comment:
We should probably have `Breadcrumbs` automatically add `aria-current` to the last `BreadcrumbsItem` automatically.

Reply:
That makes sense. I'll remove it from this example and will file a ticket for implementing this inside Breadcrumbs as well. Unless you think I can include that functionality as part of this PR. What do you think?

Reply:
A separate PR is probably best.